src/node-backend/routes/homework.js

```diff
@@ -74,6 +74,17 @@ router.post('/changeHwTaskStatus', async function (req, res) {
     }
 });
 
+// student needs help, needs {username, hwTaskId, question}
+router.post('/studentNeedsHelp', async function (req, res) {
+    try {
+        const {username, hwTaskId, question} = req.body;
+        const result = await hwController.studentHelpRequest(username, hwTaskId, question);
```

Comment:
shouldn't the request include teacher username / id?

Reply:
I thought that the teacher username would come from the task anyway, because the task has the information which teacher created/assigned it. I though it's maybe easier for frontend to just keep the task IDs?